src/components/TodoListItem.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import TodoList from "./TodoList";
+const TodoListItem = (props) => {
+  return (
+    <div>
+      <h1>{props.data}</h1>
+      <ul>
+        {props.items.map((item, i) => {
+          return <li key={i}>{item.title}</li>;
+        })}
```

Comment:
Mapping should be done in the TodoList component. The TodoListItem's concern is a single todo.